### DISCUSSION TITLE
modem-manager REV_ GUIDs

### DISCUSSION THREAD

**Discussion by lvoegl** [fwupd/fwupd#7689] [Q&A] [answered]
The `modem-manager` plugin only uses instance IDs matching `???\\VID_????&PID_????&REV_????` and `???\\VID_????&PID_????&REV_????&CARRIER_*` (see [fu-mm-device.c](https://github.com/fwupd/fwupd/blob/a893fcc15f476c805a067ca685edc2edd6a7ee1a/plugins/modem-manager/fu-mm-device.c#L156)) as _visible_ IDs that can be used to identify a device in `.metainfo.xml`. Is there a specific reason why `VID_&PID_` without `REV_` is not sufficient here and only used for quirks?

**Comment by hughsie** [comment, marked as answer]
No, I don't think so. I'd be okay with making the VID_PID ones visible I think.

**Reply by lvoegl** [reply]
Alright, I'll put this into a pull request and see what others think.
